pkg/controllermanager/controller/shoot/shoot_control_delete.go

```diff
@@ -166,7 +166,7 @@ func (c *Controller) runDeleteShootFlow(o *operation.Operation) *gardencorev1alp
 		})
 		deploySecrets = g.Add(flow.Task{
 			Name: "Deploying Shoot certificates / keys",
-			Fn:   flow.SimpleTaskFn(botanist.DeploySecrets).DoIf(!shootNamespaceInDeletion),
+			Fn:   flow.SimpleTaskFn(botanist.DeploySecrets).SkipIf(shootNamespaceInDeletion),
```

Comment:
Why is this changed?

Reply:
Isn't "Skip if" better wording for "Do If not". And probably that's the reason for having that API right?

Reply:
OK, sure, valid argument. Let's keep it. :)